example/color_cycle.py

```diff
@@ -0,0 +1,31 @@
+import matplotlib.pyplot as plt
+from diffpy.utils.parsers.loaddata import loadData
+from billinge_style.bg_plt_style import bg_plt_style
+import numpy as np
+#if use stylesheet remotely
```

Comment:
clean this.  Also, maybe add instructions how to use it, which will basically be to read the readme and install the package etc.